kickstart/scripts/admin-deploy.sh

```diff
@@ -45,13 +45,15 @@ deploy_posm_admin() {
   chmod a+rx $dst/posm-admin/scripts/*
 
   # Various scripts should be owned by other users
+  chown postgres:postgres "$dst/posm-admin/scripts/postgres_api-db-backup.sh"
   chown postgres:postgres "$dst/posm-admin/scripts/postgres_api-db-drop-create.sh"
   chown osm:osm "$dst/posm-admin/scripts/osm_api-db-init.sh"
   chown osm:osm "$dst/posm-admin/scripts/osm_api-db-populate.sh"
   chown osm:osm "$dst/posm-admin/scripts/osm_render-db-api2pbf.sh"
   chown gis:gis "$dst/posm-admin/scripts/gis_render-db-pbf2render.sh"
```

Comment:
(To be clear, the action here is to `chmod 755 "$dst/posm-admin/scripts` and remove the `chown` calls.)